saga/src/activities.ts

```diff
@@ -0,0 +1,17 @@
+import { BoundedContextClients } from './types/clients';
+
+// createActivities just returns client implementations directly right now, but later might decorate
+export function createActivities(clients: BoundedContextClients) {
```

Comment:
This is completely optional, you could just use call `proxyActivities` for each individual client class. Up to you.

Reply:
Yes you are right...when I was demoing this I was going over how activities evolve though and end up oftimes decorating something like the underlying HTTP or gRPC client (because of param serialization for example). I intentionally kept some of this ceremonial to illustrate even if it is extraneous. 

Do you want me to lean it up in this repo though?